### DISCUSSION TITLE
Cannot nodejs library in api routes

### DISCUSSION THREAD

**Discussion by ImamJanjua** [expo/expo#31688] [Archived: Questions]
So, I am using the Expo API routes in my app. I’m only using the jsonwebtoken library in the API directory, but for some reason, it's giving me an error about some dependencies like "Crypto" not being found. I mean, if the API folder runs Node.js, then it should work.
<img width="442" alt="Screenshot 2024-09-25 at 15 34 28" src="https://github.com/user-attachments/assets/1ca8cb08-1be3-489b-9332-54523932c53f">

**Comment by gonza224** [comment]
Hey, did you find a solution to this?
